dist/justified-layout.js

```diff
@@ -45,8 +45,8 @@ var Row = module.exports = function (params) {
 	this.maxAspectRatio = this.width / params.targetRowHeight * (1 + params.targetRowHeightTolerance);
 
 	// Edge case row height minimum/maximum
-	this.edgeCaseMinRowHeight = params.edgeCaseMinRowHeight || Number.NEGATIVE_INFINITY;
-	this.edgeCaseMaxRowHeight = params.edgeCaseMaxRowHeight || Number.POSITIVE_INFINITY;
```

Comment:
Can you .gitignore this dist file anyway and find out from @jeremyruppel how to make it a pre-publish or post-publish or post-install or whatever step instead? It really muddies these PRs